setup.py

```diff
@@ -17,16 +18,20 @@
 
 SHELLCHECK_VERSION = '0.7.2'
 POSTFIX_SHA256 = {
-    # TODO(rhee): detect "linux.aarch64" and "linux.armv6hf"
-    'linux': (
+    # TODO: detect "linux.armv6hf"
+    ('linux', 'aarch64'): (
+        'linux.aarch64.tar.xz',
+        'a12bdfe0f95811ad6c0a091006b919b2834b0619b460cfa596f557edd62e45ab',
+    ),
+    ('linux', 'x86_64'): (
         'linux.x86_64.tar.xz',
         '70423609f27b504d6c0c47e340f33652aea975e45f312324f2dbf91c95a3b188',
     ),
-    'darwin': (
+    ('darwin', 'x86_64'): (
```

Comment:
I think this breaks m1 macs which currently download x86_64 and use it through rosetta

Reply:
What is expected here? Adding AArch64 along with darwin will work? Please suggest. Thanks in advance.

Reply:
currently shellcheck only provides an intel binary, which should get installed if the host python is either arm or intel on m1 macs (since rosetta will run it)

Reply:
> currently shellcheck only provides an intel binary, which should get installed if the host python is either arm or intel on m1 macs (since rosetta will run it)

So, adding aarch64 along with darwin would work.

Reply:
I don't think darwin calls it aarch64 -- that's debian's naming

Reply:
Yes, I have added arm64 with darwin